src/Appwrite/SDK/Specification/Format/Swagger2.php

```diff
@@ -21,41 +25,6 @@ public function getName(): string
         return 'Swagger 2';
     }
 
-    protected function getNestedModels(Model $model, array &$usedModels): void
```

Comment:
We don't use this anywhere?

Reply:
It's common between both Swagger and OpenAPI so I moved it up to the base `Format` class